components/ide/jetbrains/image/BUILD.yaml

```diff
@@ -56,8 +55,8 @@ packages:
         SUPERVISOR_IDE_CONFIG: supervisor-ide-config_intellij.json
         JETBRAINS_BACKEND_QUALIFIER: latest
       image:
-        - ${imageRepoBase}/ide/intellij:${version}-latest
-        - ${imageRepoBase}/ide/intellij:commit-${__git_commit}-latest
+        - ${imageRepoBase}/ide/intellij-latest:${version}
```

Comment:
I think it is breaking change for Self-Hosted, since installer expects `ide/intelij`.

Can we only change version scheme? i.e. stable never publish latest as version, and latest is using prefix for a version?